Build/create-sbom.ps1

```diff
@@ -43,9 +50,14 @@ Write-Host "Using NPM project: $NpmProjectJson"
 $Solution = [System.IO.Path]::GetFullPath($Solution)
 Write-Host "Using solution file: $Solution"
 
-$OutputPath = [System.IO.Path]::GetFullPath($OutputPath)
-[System.IO.Directory]::CreateDirectory([System.IO.Path]::GetDirectoryName($OutputPath))
-Write-Host "Using output path: $OutputPath"
+$OutputFolder = [System.IO.Path]::GetFullPath($OutputFolder)
+[System.IO.Directory]::CreateDirectory($OutputFolder)
+Write-Host "Using output folder: $OutputFolder"
+
+$OutputSbomXmlProd = "$OutputFolder\re-motion.sbom.xml"
+$OutputSbomJsonProd = "$OutputFolder\re-motion.sbom.json"
+$OutputSbomXmlDev = "$OutputFolder\re-motion.development.sbom.xml"
+$OutputSbomJsonDev = "$OutputFolder\re-motion.development.sbom.json"
```

Comment:
In terms of expressiveness I think this is fine and using `Path.Combine` probably won't improve the situation. I would leave it as is but we can also change it. I don't have a firm opinion, since we won't use Linux in the near future I don't think it is going to be a problem yet